src/main/java/games/battlelore/BattleloreForwardModel.java

```diff
@@ -1,31 +1,30 @@
 package games.battlelore;
 
 import core.AbstractForwardModel;
-import core.AbstractGameData;
 import core.AbstractGameState;
 import core.actions.AbstractAction;
 import core.components.GridBoard;
-
-import core.interfaces.IGamePhase;
-import core.rules.Node;
 import games.battlelore.actions.AttackUnitsAction;
 import games.battlelore.actions.MoveUnitsAction;
 import games.battlelore.actions.PlayCommandCardAction;
 import games.battlelore.actions.SkipTurnAction;
 import games.battlelore.cards.CommandCard;
 import games.battlelore.components.MapTile;
 import games.battlelore.components.Unit;
-
 import utilities.Utils;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 public class BattleloreForwardModel extends AbstractForwardModel {
+
+    int maxTurnsToPlay = 100;
```

Comment:
This looks like it should be a parameter - but also unrelated to this PR so can be done separate, just wanted to add a quick note.

Reply:
Easy to fix at the same time. Done.